### PR TITLE
Readable JSON and YAML

### PR DESCRIPTION
**Pull request by tioirawan**
This adds readable, and minified JSON data,
and a YAML data format

Closes https://github.com/coala/gci-leaders/issues/57

### CODE REVIEW

File: lib/scrape.js [hunk 1]
```diff
@@ -127,6 +128,18 @@ async function fetchDates() {
     (a, b) => b.completed_task_instance_count - a.completed_task_instance_count
   )
 
-  fs.writeFileSync(`${__dirname}/../out/data.json`, JSON.stringify(data))
+  // readable JSON
+  fs.writeFileSync(
+    `${__dirname}/../out/data.json`,
+    JSON.stringify(data, null, '  ')
```
Comment (blazeu) [inline lib/scrape.js]: You could also put 2 instead of '  ' for clarity.

File: lib/scrape.js [hunk 2]
```diff
@@ -127,6 +128,18 @@ async function fetchDates() {
...
 
-  fs.writeFileSync(`${__dirname}/../out/data.json`, JSON.stringify(data))
+  // readable JSON
+  fs.writeFileSync(
+    `${__dirname}/../out/data.json`,
+    JSON.stringify(data, null, 2)
+  )
+  // minified JSON
+  fs.writeFileSync(
+    `${__dirname}/../out/data_minified.json`,
```
Comment (blazeu) [inline lib/scrape.js]: i think we want `data.min.json` ,  a bit shorter.

File: lib/scrape.js [hunk 3]
```diff
@@ -127,6 +128,18 @@ async function fetchDates() {
...
+  // readable JSON
+  fs.writeFileSync(
+    `${__dirname}/../out/data.json`,
+    JSON.stringify(data, null, 2)
+  )
+  // minified JSON
+  fs.writeFileSync(
+    `${__dirname}/../out/data.min.json`,
+    JSON.stringify(data)
+  )
```
Comment (andrewda) [inline lib/scrape.js]: ESLint wants this all on one line

**Review by andrewda — CHANGES_REQUESTED** [review]

File: lib/scrape.js (hunk 3 above)
Comment (blazeu) [reply lib/scrape.js]: i don't know why @gitmate-bot is not screaming at this

@jayvdb ^

**Review by andrewda — APPROVED** [review]

**Review by blazeu — APPROVED** [review]

File: lib/scrape.js (hunk 3 above)
Comment (jayvdb) [reply lib/scrape.js]: no idea

**Comment by jayvdb** [comment]
ack https://github.com/coala/gci-leaders/pull/66/commits/006c19876a6f12dc2271e2d9d6929b2ddf52cb82

File: lib/scrape.js (hunk 3 above)
Comment (blazeu) [reply lib/scrape.js]: Shit, it might be missing the eslint plugin.

**Pull request merged**